docs/custom-tests/downloading-extensions-and-tests.md

```diff
@@ -0,0 +1,124 @@
+# Downloading extensions and tests
+
+QIT automatically downloads the extension you are testing **(SUT)** and any additional plugins, dependencies, and test tags. By default, QITc fetches the latest stable versions from WooCommerce.com for paid extensions and WordPress.org for free plugins.
+
+## The extension under test
+
+The main extension you are testing must be a product in the WooCommerce.com marketplace and associated with your account. The account used during `qit connect` **must be the maintainer of this extension**. This can be a product submission or a published product.
+
+For more details, see [Authenticating with QIT](../installation-setup/authenticating.md).
+
+## Downloading paid extensions
+
+Paid extensions require authentication. Similar to the SUT, you **must be the maintainer of the paid extension** you want to include in your test. If you do not own it, you must provide a local source (for example, a ZIP file containing the extension).
+
+Below you’ll find examples of providing local sources.
+
+## Downloading free extensions
+
+Free extensions are sourced directly from WordPress.org without requiring authentication or ownership. You can still provide a local source if you want to test a modified or development version.
+
+## Additional plugins, dependencies, and test tags
+
+- **Paid (WooCommerce.com):** You must be the maintainer or provide a local source.
+- **Free (WordPress.org):** No authentication is required.
+- **Not listed in either marketplace:** A local source is required.
+- **Custom test tags**: You must own the associated extension or provide a local source.
+
+## Providing local sources
+
+In `qit.yml`:
+
+```yaml
+plugins:
+  my-plugin:
+    source: ./my-plugin.zip
+    test_tags:
+      - ./local-tests/my-plugin-tests
+```
+
+## Installing from other sources
+
+To fetch extensions from unsupported locations (such as private Git repositories), implement **custom handlers**. For details, see [Advanced Config Handlers](./../advanced-usage/advanced-config-handlers.md).
+
+Below is a refined version that incorporates all your changes and additions:
```

Comment:
I think this snuck in here unintentionally.

Reply:
Nice catch!